README.md

```diff
@@ -269,6 +269,12 @@ By default the login process is interactive, however, if that does not fit your
 
 You can switch between account profiles by using `spacectl profile select ${MY_ALIAS}`. What this does behind the scenes is point `${HOME}/.spacelift/current` to the new location. You can also delete stored credetials for a given profile by using the `spacectl profile logout ${MY_ALIAS}` command.
 
+## MCP Server
```

Comment:
Could you make the instructions more detailed? Or link to a docs page with specific instructions?

Reply:
I added some details